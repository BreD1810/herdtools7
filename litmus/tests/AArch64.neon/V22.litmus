AArch64 V22
(* Tests LD2 multiple structure *)
{
x=65535;
0:X0=x; 0:X1=y; 0:X2=z;
}
 P0                     ;
 LD2 {V0.8B,V1.8B},[X0] ;
 ST1 {V0.B}[0],[X1] ;
 ST1 {V1.B}[0],[X2] ;
forall(y=255 /\ z=255)
