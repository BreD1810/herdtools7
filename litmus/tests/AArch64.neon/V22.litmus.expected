Test V22 Required
Histogram (1 states)
1000000:>y=255; z=255;
Ok

Witnesses
Positive: 1000000, Negative: 0
Condition forall (y=255 /\ z=255) is validated
Hash=11eb898e26386d2ef4a5d09540dcb6d9
Observation V22 Always 1000000 0
